net-dns/doggo/doggo-1.0.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module shell-completion
+
+DESCRIPTION="Command-line DNS Client for Humans"
+HOMEPAGE="https://github.com/mr-karan/doggo"
+
+SRC_URI="https://github.com/mr-karan/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://www.applied-asynchrony.com/distfiles/${P}-deps.tar.xz"
```

Comment:
Please consider using my [script](https://github.com/thesamesam/sam-gentoo-scripts/blob/main/maint/bump-go) or something similar so you get a smaller and reproducible tarball.

Reply:
I spent half an hour trying to make bump-go work, and the created tarball is exactly the same size as the one created by dev-go/go-dep-tarball (which I used). In other words this isn't getting any smaller.